### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by D01044432**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/D01044432/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
```
Comment (D01044432) [inline ReadMe.md:1]: !resolved

Comment (D01044432) [inline ReadMe.md:1]: !fixed

Comment (D01044432) [inline ReadMe.md:1]: !fix

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
```
Comment (D01044432) [inline ReadMe.md:2]: deafffff

**Pull request merged**